crates/holochain_conductor_api/src/config/conductor.rs

```diff
@@ -76,6 +76,10 @@ pub struct ConductorConfig {
     /// that this value matches the one in KitsuneP2pConfig!
     #[serde(default)]
     pub tracing_scope: Option<String>,
+
+    /// Tuning parameters to adjust the behaviour of the conductor.
+    #[serde(default)]
+    pub tuning_params: Option<ConductorTuningParams>,
```

Comment:
This is a breaking change. I suggest we don't back-port this part. Instead I'll modify the tests that were relying on this on 0.2 to have longer timeouts.

Reply:
Is it breaking? The serde default should handle configs that don't specify this.

Reply:
It is unfortunately, because of the conductor config being created in code. It will affect the Launcher I believe and possibly the Holochain runner. It wouldn't affect most users as you say, because of serde